src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1303,6 +1303,28 @@ class NodeConfig(object):
         pbs.logmsg(pbs.EVENT_DEBUG4, '%s: %s' % (caller_name(), numa_nodes))
         return numa_nodes
 
+    def _devinfo(self, path):
+        """ Returns major minor and type from device """
+        # If the stat fails, log it and continue.
+        try:
+            statinfo = os.stat(path)
+        except KeyboardInterrupt:
+            raise
+        except:
+            pbs.logmsg(pbs.EVENT_DEBUG2, '%s: Stat error on %s' %
+                       (caller_name(), path))
+            return None
```

Comment:
You know what exception stat() will raise.  check for that.  Don't do a blanket except.  Then you don't need to have the check for KeyboardInterrupt.